lib/orchestrator/application.ex

```diff
@@ -104,30 +101,13 @@ defmodule Orchestrator.Application do
 if Mix.env() == :test do
   # For now, the simplest way to make tests just do tests, not configure/start anything.
   defp filter_children(_children), do: []
-  defp configure_api_token, do: :ok
-  defp configure_slack_reporter, do: :ok
 else
-  defp filter_children(children), do: children
-  defp configure_api_token do
-    token =
-      case System.get_env("CANARY_API_TOKEN") do
-        nil ->
-          "fake-token-for-dev"
-
-        token ->
-          Orchestrator.Configuration.translate_value(token)
-      end
-
-    Application.put_env(:orchestrator, :api_token, token)
-  end
-  defp configure_slack_reporter do
-    token = System.get_env("SLACK_API_TOKEN")
-    |> Orchestrator.Configuration.translate_value()
-    Application.put_env(:slack, :api_token, token)
-
-    channel = System.get_env("SLACK_ALERTING_CHANNEL")
-    |> Orchestrator.Configuration.translate_value()
-    Application.put_env(:slack, :reporting_channel, channel)
+  defp filter_children(children) do
```

Comment:
Isn't this a bit "dirty" versus the other way around, only adding the module if we need it?

Reply:
Added  a commit to only add the HostTelemetry module if it's enabled in this commit https://github.com/Metrist-Software/orchestrator/pull/114/commits/a6aa4b25eba46ba8e1ad07045a16634a9f67d17a